test/loader_test.rb

```diff
@@ -45,6 +45,15 @@
         CoffeeShop::Controller.configuration.default_response_format.must_equal(:html)
       end
 
+      it 'configures controller to use default headers' do
+        CoffeeShop::Controller.configuration.default_headers.
+          must_equal({
+            "X-Frame-Options" => "DENY",
+            "Content-Security-Policy" => "form-action 'self'; referrer origin-when-cross-origin; reflected-xss block; manifest-src 'self'; frame-ancestors 'self'; base-uri 'self'; default-src 'none'; connect-src 'self'; img-src 'self'; style-src 'self'; font-src 'self'; object-src 'self'; plugin-types application/pdf application/x-shockwave-flash application/x-java-applet; child-src 'self'; media-src 'self'",
```

Comment:
@joneslee85 I was referring to CSP vs CSPRO debate. :wink: 

For multiline strings my idea is to transform the given string into a CSP compatible one. That means to change how [`Hanami::Config::Security#content_security_policy`](https://github.com/hanami/hanami/blob/master/lib/hanami/config/security.rb#L49) works

```ruby
  def content_security_policy(value = nil)
        if value.nil?
          @content_security_policy
        else
          @content_security_policy = value # TRANSFORM THE INPUT HERE
        end
      end
```